pkg/sql2pgroll/alter_table.go

```diff
@@ -443,6 +430,17 @@ func canConvertColumnForSetDataType(column *pgq.ColumnDef) bool {
 	return true
 }
 
+func getQualifiedRelationName(rel interface {
+	GetSchemaname() string
+	GetRelname() string
+},
```

Comment:
Do we need the generality of the interface here? All the call sites pass in a `*pgq.RangeVar` - maybe the parameter could be of type `*pgq.RangeVar` instead?

Reply:
Ah, yeah, good point. I'll make those changes.